MoneyDataType/LegacyAmountConverter.cs

```diff
@@ -21,7 +21,7 @@ public override Amount ReadJson(Newtonsoft.Json.JsonReader reader, Type objectTy
             string symbol = null;
             string iso = null;
             int? dec = null;
-            bool unknown = false;
+            //bool unknown = false;
```

Comment:
Don't comment out code. If it must be deleted, delete it.

Reply:
Commented out lines are deleted.